### ISSUE TITLE
Income Expense radiobuttons in new transaction dialog have reversed tab order

### ISSUE THREAD

**Issue by jm6271** [jm6271/MyMoney#254]
The tab order is backwards for the Income and Expense radiobuttons in the new transaction dialog. Their tab order should be reversed.
